Cargo.toml

```diff
@@ -126,12 +126,18 @@ opt-level = 3
 [profile.release]
 panic = "abort"
 lto = true
+# https://github.com/rust-lang/rust/issues/47745
```

Comment:
Could you add a small text?

```suggestion
# codegen-units set to 1 to avoid performance regressions when combined with LTO
# See https://github.com/rust-lang/rust/issues/47745
```

Reply:
I just applied your suggestion.